interpreters/src/show.rs

```diff
@@ -0,0 +1,152 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+use std::{convert::TryInto, sync::Arc};
+
+use arrow_deps::arrow::{
+    array::StringArray,
+    datatypes::{DataType, Field, Schema},
+    record_batch::RecordBatch,
+};
+use async_trait::async_trait;
+use catalog::manager::Manager;
+use snafu::{Backtrace, ResultExt, Snafu};
+use sql::{
+    ast::ShowCreateObject,
+    plan::{ShowCreatePlan, ShowPlan},
+};
+
+use crate::{
+    context::Context,
+    create::{self, get_catalog, get_schema},
+    interpreter::{
+        Interpreter, InterpreterPtr, Output, Result as InterpreterResult, ShowCreateTable,
+        ShowDatabases, ShowTables,
+    },
+    show_create::ShowCreateInterpreter,
+};
+
+const SHOW_TABLES_SCHEMA: &str = "Tables";
+const SHOW_DATABASES_SCHEMA: &str = "Schemas";
+
+#[derive(Debug, Snafu)]
+#[snafu(visibility(pub(crate)))]
+pub enum Error {
+    #[snafu(display(
+        "Unsupported show create type, type: {:?}, err:{}",
+        obj_type,
+        backtrace
+    ))]
+    UnsupportedType {
+        obj_type: ShowCreateObject,
+        backtrace: Backtrace,
+    },
+
+    #[snafu(display("Create a new arrow RecordBatch fail, err:{}", source))]
+    NewRecordBatch {
```

Comment:
Maybe `CreateRecordBatch` is better because `New` in this word seems an adjective rather a verb.